pkg/vz/rosetta_directory_share.go

```diff
@@ -0,0 +1,12 @@
+//go:build darwin && !arm64
+// +build darwin,!arm64
```

Comment:
I believe we need to use !no_vz buildtag in both rosetta files to avoid this issue 
https://github.com/lima-vm/lima/issues/1176

Reply:
Done